assembly/assembly-wsmaster-war/src/test/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,63 @@
+
+    Contributors:
+      Codenvy, S.A. - initial API and implementation
+
+-->
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence persistence_1_0.xsd" version="1.0">
+    <persistence-unit name="test" transaction-type="RESOURCE_LOCAL">
+        <provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>
+        <class>org.eclipse.che.api.user.server.model.impl.UserImpl</class>
+        <class>org.eclipse.che.api.user.server.model.impl.ProfileImpl</class>
+        <class>org.eclipse.che.api.user.server.jpa.PreferenceEntity</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.WorkspaceImpl</class>
+        <class>org.eclipse.che.api.machine.server.model.impl.SnapshotImpl</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.WorkspaceImpl</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.WorkspaceConfigImpl</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.ProjectConfigImpl</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.EnvironmentImpl</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.ProjectConfigImpl$Attribute</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.SourceStorageImpl</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.stack.StackImpl</class>
+        <class>org.eclipse.che.api.machine.server.model.impl.MachineConfigImpl</class>
+        <class>org.eclipse.che.api.machine.server.model.impl.CommandImpl</class>
+        <class>org.eclipse.che.api.machine.server.model.impl.AclEntryImpl</class>
+        <class>org.eclipse.che.api.machine.server.model.impl.SnapshotImpl</class>
+        <class>org.eclipse.che.api.user.server.model.impl.UserImpl</class>
+        <class>org.eclipse.che.api.ssh.server.model.impl.SshPairImpl</class>
+        <class>org.eclipse.che.api.factory.server.model.impl.ActionImpl</class>
+        <class>org.eclipse.che.api.factory.server.model.impl.AuthorImpl</class>
+        <class>org.eclipse.che.api.factory.server.model.impl.ButtonAttributesImpl</class>
+        <class>org.eclipse.che.api.factory.server.model.impl.ButtonImpl</class>
+        <class>org.eclipse.che.api.factory.server.model.impl.FactoryImpl</class>
+        <class>org.eclipse.che.api.factory.server.model.impl.IdeImpl</class>
+        <class>org.eclipse.che.api.factory.server.model.impl.OnAppClosedImpl</class>
+        <class>org.eclipse.che.api.factory.server.model.impl.OnProjectsLoadedImpl</class>
+        <class>org.eclipse.che.api.factory.server.model.impl.OnAppLoadedImpl</class>
+        <class>org.eclipse.che.api.factory.server.model.impl.PoliciesImpl</class>
+        <class>org.eclipse.che.api.factory.server.FactoryImage</class>
+        <!--<class>org.eclipse.che.api.factory.server.model.impl.AuthorImpl</class>-->
```

Comment:
I think you can remove this commented line because you already have AuthorImpl here https://github.com/eclipse/che/pull/2031/files#diff-2aebeda36e4acb8637396901436b9dd2R37